### DISCUSSION TITLE
Time zone change in kong logs

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8743] [Help]
Hi,

I am getting UTC timezone in kong logs, I want to make it as IST.
Can some one please help.
Below is my log configuration

  KONG_NGINX_PROXY_SET_BY_LUA: $$start_time 'return ngx.now()'
 KONG_NGINX_HTTP_LOG_FORMAT: show_everything '$proxy_protocol_addr -$remote_addr - $remote_user [$time_local] "$request" $status ($request_time s)'

**Comment by ADD-SP** [comment]
```log
2022/04/20 10:44:01 [debug] 471862#0: free: 0000000000000000
```

Hi, above is a Nginx log with a formatted time at the beginning. If you wish to change this time you can set the environment variable `TZ` to the specified time zone. 

https://github.com/nginxinc/docker-nginx/issues/149

**Reply by guptadeepali39** [reply]
Hi I am using kong helm chart and have set the env variables as below:
env:
  TZ: Asia/Kolkata

Still showing UTc time zone only

**Reply by windmgc** [reply]
One thing I want to remind here is that if you're running Kong in a container environment, usually the `/etc/locatime` and `/usr/share/timezone` are missing because of image size considerations, which means that even if you're setting the TZ environment it'll not work as expected. Please read the issue https://github.com/nginxinc/docker-nginx/issues/149 carefully again, you'll find a workaround in the following link https://github.com/gliderlabs/docker-alpine/issues/136#issuecomment-217809326 (which can help you add back the timezone info and local timezone configs)

If you're running images like `alpine` or `debian` the timezone package are both called `tzdata`, AFAIK
